2024-10-08 16:48:56,854 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Configure stats pid to 1464688
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Loading settings from /home/ah5087/.config/wandb/settings
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Loading settings from /home/ah5087/COS397_IW/model/wandb/settings
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'model/train.py', 'program_abspath': '/home/ah5087/COS397_IW/model/train.py', 'program': 'train.py'}
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_init.py:_log_setup():532] Logging user logs to /home/ah5087/COS397_IW/model/wandb/run-20241008_164856-4kmsmuyb/logs/debug.log
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_init.py:_log_setup():533] Logging internal logs to /home/ah5087/COS397_IW/model/wandb/run-20241008_164856-4kmsmuyb/logs/debug-internal.log
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_init.py:init():617] calling init triggers
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {'obs_dim': 20402, 'action_dim': 2, 'latent_dim': 5, 'num_episodes': 1000}
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_init.py:init():667] starting backend
2024-10-08 16:48:56,855 INFO    MainThread:1464688 [wandb_init.py:init():671] sending inform_init request
2024-10-08 16:48:56,856 INFO    MainThread:1464688 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-08 16:48:56,856 INFO    MainThread:1464688 [wandb_init.py:init():684] backend started and connected
2024-10-08 16:48:56,859 INFO    MainThread:1464688 [wandb_init.py:init():779] updated telemetry
2024-10-08 16:48:56,890 INFO    MainThread:1464688 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-08 16:50:26,916 ERROR   MainThread:1464688 [wandb_init.py:init():838] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-08 16:50:26,916 ERROR   MainThread:1464688 [wandb_init.py:init():1262] error in wandb.init()
Traceback (most recent call last):
  File "/home/ah5087/.pyenv/versions/myenv/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/home/ah5087/.pyenv/versions/myenv/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 844, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-08 16:50:26,935 WARNING MsgRouterThr:1464688 [router.py:message_loop():77] message_loop has been closed
